bin/run-in-env

```diff
@@ -34,17 +40,53 @@ def run_command_with_env(config_map, command)
   exec command
 end
 
-def pull_config_from_aws(env)
-  param_name = "/config/#{env}"
-  @log.debug('Fetching parameter ' + param_name)
+def pull_ssm_parameters(parameters)
+  @log.debug 'Fetching parameters ' + parameters.to_s
```

Comment:
should this be removed? not sure if GitHub actions are publicly accessible, don't want debug info to be exposed?

Reply:
This script doesn't run in github actions and it's only when you request verbose, so there is no risk here.